LiskHQ/lisk-desktop
# Implement redesign of send component - Closes #2

### What was the problem?
Need to implement the redesign of the send component

### How did I fix it?
For now, since this task is depending on the multiple step component, I implemented the parts like this:
In the transaction components we have transactions and `Transfer`. In `Transfer` I will either render `Send` or `Receive` on button click. In `Transfer` and `Send` I distinguish between the following states: isEditable, which means the form is either writable or only readable and `transactionIsSuccessful` for the success/failure messages. In the next ticket I will work more on building a proper modal for this. 

### How to test it?
- Compare with the designs
- Try to send money and make sure you get a success message
- Make sure the button is disabled after confirming the transaction
- Make sure all other components still work as expected (e.h. Dialog)

Note: 
what still needs to be done:

### Review checklist
- [x] The PR solves https://github.com/LiskHQ/lisk-wallet/issues/2
